packages/lwc-language-server/src/metadata-utils/__tests__/util.ts

```diff
@@ -24,6 +25,8 @@ export async function validate(
     await indexer(context, true);
     const path = join(workspacePath, '.sfdx', 'typings', 'lwc', expectedTypeDeclarationFileName);
     expect(path).toExist();
-    const contents = fs.readFileSync(path, 'utf8');
-    expect(contents).toBe(expectedTypeDeclarations);
+    // For windows we need to normalize line endings, we do that using eol.
+    const contents = eol.auto(fs.readFileSync(path, 'utf8'));
```

Comment:
seems weird to me that we have to use this eol thing. is it a problem we can fix with the file itself?

Reply:
As soon as someone goes and edits the file again, it'll all blow up. Line endings are a pain in the butt.

Reply:
I think this is due to the workspace getting restored from cache, which would cause git to have checked out the filesystem with the wrong style line endings

Reply:
we'll remove this later, just wanna get this PR in